src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,10 @@
+$days-of-the-week:
+  "mon" 0,
+  "tue" (101px),
+  "wed" (101px * 2),
+  "thu" (101px * 3),
+  "fri" (101px * 4),
+  "sat" (101px * 5),
+  "sun" (101px * 6);
```

Comment:
what if gap will changed? it will be not 101px more. maybe you can use function for this?